oci/oci.go

```diff
@@ -474,7 +474,10 @@ func (r *Runtime) StopContainer(c *Container) error {
 func (r *Runtime) DeleteContainer(c *Container) error {
 	c.opLock.Lock()
 	defer c.opLock.Unlock()
-	return utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.Path(c), "delete", c.name)
+	if _, err := utils.ExecCmd(r.Path(c), "delete", c.name); err != nil && !strings.Contains(err.Error(), "does not exist") {
```

Comment:
This is `runc` specific, and also specific to the current `runc` implementation. Could we have a more generic implementation where we ping the container process when getting an error ? If the container process is not there, then we ignore the error and return nil ?